src/api/data/cache/cache_test.go

```diff
@@ -90,3 +74,17 @@ func TestCachedChartsRefresh(t *testing.T) {
 	err := chartsImplementation.Refresh()
 	assert.NoErr(t, err)
 }
+
+func getChartsImplementation() data.Charts {
```

Comment:
I am wondering if we should stub these repositories (removing the external dependency) so we remove the possibility of getting different test results.

It should speed up the tests as well no?